lib/prolongator_mma.in.cu

```diff
@@ -0,0 +1,236 @@
+#include <color_spinor_field.h>
+#include <multigrid.h>
+#include <tunable_nd.h>
+#include <expand_list.hpp>
+
+#ifdef QUDA_MMA_AVAILABLE
+#include <device.hpp>
+#include <kernels/prolongator_mma.cuh>
+#include <mma_tensor_op/smma_m16n8k8_sm70.cuh>
+#endif
+
+namespace quda
+{
+
+#ifdef QUDA_MMA_AVAILABLE
+
+  template <typename Float, typename vFloat, int fineSpin, int fineColor, int coarseSpin, int coarseColor, int nVec>
+  class ProlongateLaunchMma : public TunableKernel
+  {
+
+    ColorSpinorField &out;
+    const ColorSpinorField &in;
+    const ColorSpinorField &V;
+    const int *fine_to_coarse;
+    int parity;
+    QudaFieldLocation location;
+
+    using mma_t = typename mma::mg_mma_prolongator_t<Float>::type;
+
+    static constexpr int spin_block_factor = spin_mapper<fineSpin, coarseSpin>::get_spin_block_factor();
+
+    static constexpr int m = nVec;
+    static constexpr int n = fineColor * spin_block_factor;
+    static constexpr int k = coarseColor;
+
+    static constexpr int n_atom_size = mma_t::MMA_N;
+    static constexpr int m_atom_size = mma_t::MMA_M;
+    static constexpr int k_atom_size = mma_t::MMA_K;
+    static constexpr int block_atom_size = 32 / 8;
```

Comment:
A comment about why the numerical value block_atom_size = 32 / 8 would be helpful

Reply:
https://github.com/lattice/quda/pull/1497/commits/87d332f5178d97dc15e642d362f1e5d4e1f2bfd8